README.md

```diff
@@ -28,6 +28,15 @@ Questions, bug reporting and issue tracking are provided by GitHub. Please
 report all bugs by creating a new issue with the bug tag. You can ask
 questions by creating a new issue with the question tag.
 
+Contributing
+------------
+
+Contributing to DTK is easy. Just send us a [pull request](https://help.github.com/articles/using-pull-requests/).
```

Comment:
I feel like it is a much higher barrier. I would lower the requirements for PR, but once PR is submitted encourage the submitter to provide the tests.